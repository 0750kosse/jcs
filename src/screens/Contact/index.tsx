import React from 'react';
import Info from '../../components/Info'
import Form from '../../components/Form'
import { contact } from '../../lib/data'

export default () => {
  return (
    <>
      <Info
        title={contact.title}
        description={contact.description} />
      <Form />
    </>

  )
}