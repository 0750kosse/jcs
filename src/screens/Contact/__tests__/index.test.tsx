import React from 'react'
import Contact from '../index'
import { render } from '@testing-library/react'

describe('Contact screen', () => {
  it('should render correclty', () => {
    const wrapper = render(<Contact />)
    expect(wrapper).toMatchSnapshot()
  })
})