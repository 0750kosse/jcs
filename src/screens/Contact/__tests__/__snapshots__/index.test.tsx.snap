// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact screen should render correclty 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bdfBwQ gQQVoj"
      >
        <h1
          class="sc-gsTCUz"
        >
          Get in touch
        </h1>
        <p
          class="sc-dlfnbm gquITC"
        >
          It will be great to hear from you! Feel free to contact me for questions, or just to say hola!
        </p>
      </div>
      <form
        class="sc-hKgILt iKfnOl contact-form"
        id="contact"
      >
        <label
          class="sc-eCssSg visuallyhidden"
          for="name"
        />
        <input
          class="sc-jSgupP fhpxRR"
          id="name"
          name="user_name"
          placeholder="Name..."
          required=""
          type="text"
        />
        <label
          class="sc-eCssSg visuallyhidden"
          for="email"
        />
        <input
          aria-describedby="email-address"
          aria-invalid="true"
          class="sc-jSgupP fhpxRR"
          id="email"
          name="user_email"
          placeholder="Email..."
          required=""
          type="text"
        />
        <label
          class="sc-eCssSg visuallyhidden"
          for="message"
        />
        <textarea
          class="sc-gKsewC ciGxfK"
          id="message"
          name="message"
          placeholder="Leave your message..."
          required=""
        />
        <input
          class="sc-jSgupP fhpxRR"
          type="submit"
          value="Send message"
        />
      </form>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bdfBwQ gQQVoj"
    >
      <h1
        class="sc-gsTCUz"
      >
        Get in touch
      </h1>
      <p
        class="sc-dlfnbm gquITC"
      >
        It will be great to hear from you! Feel free to contact me for questions, or just to say hola!
      </p>
    </div>
    <form
      class="sc-hKgILt iKfnOl contact-form"
      id="contact"
    >
      <label
        class="sc-eCssSg visuallyhidden"
        for="name"
      />
      <input
        class="sc-jSgupP fhpxRR"
        id="name"
        name="user_name"
        placeholder="Name..."
        required=""
        type="text"
      />
      <label
        class="sc-eCssSg visuallyhidden"
        for="email"
      />
      <input
        aria-describedby="email-address"
        aria-invalid="true"
        class="sc-jSgupP fhpxRR"
        id="email"
        name="user_email"
        placeholder="Email..."
        required=""
        type="text"
      />
      <label
        class="sc-eCssSg visuallyhidden"
        for="message"
      />
      <textarea
        class="sc-gKsewC ciGxfK"
        id="message"
        name="message"
        placeholder="Leave your message..."
        required=""
      />
      <input
        class="sc-jSgupP fhpxRR"
        type="submit"
        value="Send message"
      />
    </form>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
